woocommerce/woocommerce
# Disable the new navigation in the WC Admin.

### All Submissions:






### Changes proposed in this Pull Request:


Disable the new navigation. The feature development is currently on hold. 

Closes #35157 .






### How to test the changes in this Pull Request:



1. Navigate to `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features` and verify that the checkbox is disabled.



### Other information:




### FOR PR REVIEWER ONLY:
